sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/HttpPostControllerActionShouldValidateInput.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Web.Mvc;
+
+namespace Tests.Diagnostics
+{
+    public class FooBarController1 : Controller
+    {
+        [HttpPost]
+        [ValidateInput(false)] // Noncompliant
```

Comment:
Why not check that the squigglie is reported on the expected characters?

Reply:
Cause I went too fast and forgot... Thx

Reply:
Done.